services/transactionpool/validate_transation.go

```diff
@@ -0,0 +1,13 @@
+package transactionpool
+
+import (
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+)
+
+func (s *service) validateTransaction(input *protocol.SignedTransaction) (bool, error) {
```

Comment:
Let's make a rule that these external files are no longer receivers, they are static functions that receive all of their dependencies (like VM) in their arguments